src/pages/ConnectWallet/ConnectWallet.tsx

```diff
@@ -178,83 +160,62 @@ export const ConnectWallet = () => {
             maxWidth={maxWidth}
             width={width}
           >
-            {isMobile ? (
```

Comment:
This logical branch could never be entered, as if `isMobile` was `true` we would have early exited above.